src/views/imaged-object-editor/imaged-object-editor.vue

```diff
@@ -365,10 +367,17 @@ export default class ImagedObjectEditor
         // this.waiting = true;
         (this.$refs.newArtefactName as any).focus();
     }
-    private async mounted() {
+
+   // moved code to created()
+   // from  private async mounted() {
+   // in order to have the masterImage ready before mounted
+   // for the <zoomed...><svg> ... part
```

Comment:
When using async created() I think mounted() is called immediately after created() without any await.  Unless they have changed things to support async lifecycle.  It is your variable `waiting` that keeps the template from blowing up due all sorts of missing data, `masterImage` is definitely not ready by the time mounted() gets called.